docs/tools/identity/self-id.md

```diff
@@ -1,30 +1,35 @@
-# Self.ID (coming soon)
+# Self.ID
 
-Self.ID is a web application that allows users to manage their basic, profile-related IDX records.
+[Self.ID](https://self.id) is a web application that provides a UI for basic identity functionality for [3ID DIDs](../../authentication/3id-did/method.md). Any project that wants their users to have basic identity functionality without wanting to integrate editing features directly into their UI can instead direct users to Self.ID. With Self.ID, users havs a simple interface for managing their core identity information, while third-party applications can query this data using the [IDX SDK](./idx.md).
 
-## Usage
+## **Applications**
 
-## Design
+- **Self.ID on mainnet**: https://self.id
+- **Self.ID on Clay Testnet**: https://clay.self.id
 
-Self.ID integrates with the following standards, tools and services.
+!!! warning ""
 
-### Authentication
+    Self.ID works, but is under active development. Please report issues in the #self-id channel in the [Ceramic Discord](https://chat.ceramic.network) server.
 
-3ID Connect: Uses 3ID DID method and allows users to authenticate with their existing blockchain wallets.
+## **Features**
 
-### Identity
+**Login with blockchain wallets**: Self.ID integrates with the [3ID Connect SDK](../../authentication/3id-did/3id-connect.md) for [authentication](../../learn/build/authentication.md) to allow users to create and use a [3ID DID](../../authentication/3id-did/method.md) from one or more existing blockchain wallets.
 
-IDX:
+**Basic profiles**: Self.ID allows users to create and/or edit a basic profile for their 3ID DID. To achieve this, Self.ID integrates with the [IDX SDK](./idx.md) to create and query [IDX records](./idx.md#records) that conform to the [Basic Profile definition and schema (CIP-19)](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-19/CIP-19.md).
 
-IdentityLink:
+**Link Web3 accounts**: Self.ID allows users to link many different blockchain accounts to their 3ID. To achieve this, Self.ID integrates with the [IDX SDK](./idx.md) to create and query [IDX records](./idx.md#records) that conform to the [Crypto Accounts definition and schema (CIP-21)](https://github.com/ceramicnetwork/CIP/blob/main/CIPs/CIP-21/CIP-21.md). Each user's Crypto Accounts record in IDX contains a list of [streamIDs](../../learn/glossary.md#streamid) for the [Caip10Links](../../streamtypes/caip-10-link/overview.md) of their 3ID.

```

Comment:
We don't have this directly in self.id at the moment so maybe we can leave this section out until it's implemented?

Reply:
We kind of provide this through 3ID Connect. Maybe i could make that more clear. Think that would address the concern?

Reply:
I don't think the flow is obvious now, especially coming from self.id that might require "logging out" and connecting again via 3ID Connect, so it's not really user-friendly at this point.

Reply:
Okay removed for now